gulpfile.js

```diff
@@ -1,97 +1,84 @@
 // Gulpfile to build sonoma CLI
 
-const clean = require('gulp-clean');
-const gulp = require('gulp');
-const rimraf = require('rimraf');
-const sourcemaps = require('gulp-sourcemaps');
-const ts = require('gulp-typescript');
-const autocompleteTree = require('./scripts/autocomplete-tree');
+const clean = require("gulp-clean");
+const gulp = require("gulp");
+const rimraf = require("rimraf");
+const sourcemaps = require("gulp-sourcemaps");
+const ts = require("gulp-typescript");
+const autocompleteTree = require("./scripts/autocomplete-tree");
 
-let tsProject = ts.createProject('tsconfig.json');
+const tsProject = ts.createProject("tsconfig.json");
```

Comment:
maybe those can use import syntax? just wondering

Reply:
I assume you're referring to the `require` statements above. I read up on this syntax a bit. It should work, but doesn't. I'm getting unexpected token errors on the asterisk. Maybe we're not using ES6. I'll leave this for this PR.